Integrations/Gmail/Gmail.py

```diff
@@ -1382,35 +1378,31 @@ def transient_attachments(transientFile, transientFileContent, transientFileCID)
     if transientFile is None or len(transientFile) == 0:
         return []
 
+    if transientFileContent is None:
+        transientFileContent = []
+
+    if transientFileCID is None:
+        transientFileCID = []
+
     attachments = []
-    entry_number = 0
-    for file_name in transientFile:
+    for triplet in it.izip_longest(transientFile, transientFileContent, transientFileCID):
```

Comment:
also notice that you are changing the default here from empty string to `None` for `file_data` and `file_cid`, is that ok?
```suggestion
    for file_name, file_data, file_cid in it.izip_longest(transientFile, transientFileContent, transientFileCID):
```